install/linux/docker-ce/debian.md

```diff
@@ -164,6 +164,7 @@ from the repository.
 
     To also add the **edge** repository, add `edge` after `stable` on the last
     line of the command.
+    To add an unsupported operating Debian-based operating system, you will need to know which distribution you have, such as `jessie`.
```

Comment:
If a distribution is not supported we should not even talk about it in the docs or provide workarounds. So can you remove this line before we can merge this contribution?

Whenever the docs provide workarounds for unsupported configs, users are confused on whether something is supported or not and will end up deploying an unsupported config. then when something breaks they will be completely disappointed with the product, even though the product never supported that config.

We had situations like this in the past, and I would like to avoid that at all costs.